addons/mass_mailing/data/mailing_data_templates.xml

```diff
@@ -162,7 +162,7 @@
             <tr>
                 <td style="margin: 0; padding:0;">
                     <table cellspacing="0" cellpadding="0" border="0" align="center" style="width:100%; border-collapse: collapse;">
-                        <tr t-att-style="'color: ' + (company.secondary_color or  '#875a7b') + '; font-size: 16px; font-weight: 500;'">
+                        <tr t-att-style="'color: ' + (company.email_secondary_color or  '#875a7b') + '; font-size: 16px; font-weight: 500;'">
```

Comment:
(Same question about upgrade script).

Bonus: "black" is still unhappy on upgrade PR.